hack/rapture/publish-packages.sh

```diff
@@ -68,7 +68,7 @@ download_packages() {
   for package in "${!locations[@]}"; do
     rm -rf "${locations[$package]}" || true
     mkdir -p "${locations[$package]}"
-    gsutil -m cp -r "$bucket/v$version/$package/*" "${locations[$package]}"
+    curl -L "$bucket/v$version/$package/*" -o "${locations[$package]}"
```

Comment:
Using a wildcard will not work when downloading through https, my suggestion would be to wait until we [record them in the provenance attestation](https://github.com/kubernetes/release/issues/3065) and use that to download them instead.